src/main/java/edu/kis/powp/jobs2d/command/OperateToCommand.java

```diff
@@ -34,4 +34,20 @@ public Object clone() {
 	public void accept(VisitorCommand visitor) {
 		visitor.visit(this);
 	}
+
+	public int getPosX() {
+		return posX;
+	}
+
+	public void setPosX(int posX) {
+		this.posX = posX;
+	}
```

Comment:
Forcing mutability isn't always the best option.

Reply:
I don't really see another way here due to the current architecture.
I've added an interface that should somehow manage the issue.